app/controllers/content_items_controller.rb

```diff
@@ -45,7 +45,11 @@ def special_route?(content_item)
   end
 
   def present(content_item)
-    presenter_name = content_item['schema_name'].classify + 'Presenter'
+    presenter_name = if service_sign_in_format?(content_item["schema_name"])
```

Comment:
Maybe extract this out into a separate method.

Reply:
Good point. Will extract into a method and fix up.

Reply:
@thomasleese Amended and fixed up